test/ERC6551/ERC6551RegistryTest.js

```diff
@@ -0,0 +1,200 @@
+const {
```

Comment:
Can these tests be moved to typescript instead? And are they triggered by npm test?